recipes/plantcv/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "plantcv" %}
+{% set version = "3.9.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a00570af93835582135d869f34b87142415149922e05119bb1ed2ac7fe7f07a0
+
+build:
+  number: 0
+  noarch: python
```

Comment:
This is not a noarch package I think

Reply:
I'm not sure why the linter suggests this based on "selectors" since there are no selectors specified in the recipe. It also seems to think the example recipe was modified even though it's not.

But otherwise, we do build this as noarch on PyPI and currently on Bioconda. OpenCV is the one tricky dependency because it relies on some external libraries. Not sure if that makes this package not noarch?

Reply:
There are selectors in the recipe. All of the `[linux]` phrases are selectors

Reply:
Oops, I totally overlooked that. Thanks! I had included these packages in the build and host requirements because I though OpenCV needed them, but I removed them now because I realized that `yum_requirements.txt` takes care of the runtime requirements for libGL. We don't link against these directly in our package.